signer/src/transaction_signer.rs

```diff
@@ -625,28 +625,38 @@ where
     }
 
     #[tracing::instrument(skip(self))]
-    async fn handle_pending_withdraw_request(
+    async fn handle_pending_withdrawal_request(
         &mut self,
-        withdraw_request: model::WithdrawalRequest,
+        withdrawal_request: model::WithdrawalRequest,
+        bitcoin_chain_tip: &model::BitcoinBlockHash,
     ) -> Result<(), Error> {
         // TODO: Do we want to do this on the sender address or the
         // recipient address?
         let is_accepted = self
-            .can_accept(&withdraw_request.sender_address.to_string())
+            .can_accept(&withdrawal_request.sender_address.to_string())
             .await;
 
+        let msg = message::SignerWithdrawalDecision {
+            request_id: withdrawal_request.request_id,
+            block_hash: withdrawal_request.block_hash.0,
```

Comment:
Plz double-check that this message is (1) correct, (2) populated with the correct data